include/onnxruntime/core/platform/EigenNonBlockingThreadPool.h

```diff
@@ -712,6 +712,7 @@ class ThreadPoolTempl : public onnxruntime::concurrency::ExtendedThreadPoolInter
         num_threads_(num_threads),
         allow_spinning_(allow_spinning),
         set_denormal_as_zero_(thread_options.set_denormal_as_zero),
+        enable_good_worker_hints_(thread_options.enable_good_worker_hints),
```

Comment:
Having options like these are useful for our perf testing (avoiding the need to rebuild, upload new images to Polaris / Anubis / etc), but can we remove the flag prior to merging to the main branch.

Suggest we retain GoodWorkerHints in this PR, and then remove in the "sticky" PR.

Generally we want to keep a small number of configuration options, mapped to user-visible features such as the CPU vs perf trade-off in allow_spinning.

Reply:
Reverted, let's do it in sticky PR.